infrastructure/zk/src/init.ts

```diff
@@ -32,6 +32,7 @@ export async function init(initArgs: InitArgs = DEFAULT_ARGS) {
     let envFileContent = fs.readFileSync(process.env.ENV_FILE!).toString();
     envFileContent += `VALIDIUM_MODE=${validiumMode}\n`;
     fs.writeFileSync(process.env.ENV_FILE!, envFileContent);
+    process.env.VALIDIUM_MODE = validiumMode.toString();
```

Comment:
@popzxc what's the right way to do this now?

Reply:
The right way is to have a proper configuration option (e.g. an entry in `zksync_config`, if it's a separate struct then also an impl in `zksync_env_config`, and entry in `etc/env/base/xxx.toml`). We should not edit env file and `process.env` manually here.